app/lambda/get_credentials/lambda_function.py

```diff
@@ -0,0 +1,39 @@
+import os
+import json
+import sys
+
+sys.path.append("../")
+from lambda_base import LambdaBase
+from helper import AwsHelper
+
+
+class GetCredentialsLambda(LambdaBase):
+    def __init__(self): # implementation-specific args and/or kwargs
+        pass
+
+    def handle(self, event, context):
+        print("event: {}".format(event))
+        sts = AwsHelper().getResource('sts')
```

Comment:
This needs to be changed to be getClient instead of getResource

Reply:
Done. https://github.com/aws-samples/medical-transcription-analysis/blob/691348b78d89ba8652fa177e9a002b4466b0ef6b/app/lambda/get_credentials/lambda_function.py#L14-L17